lib/chef/telemetry.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "log"
+require "time" unless defined?(Time.zone_offset)
+require "chef-licensing"
+require_relative "telemetry/null"
+require_relative "telemetry/http"
+
+class Chef
+  class Telemetry
+
+    @@instance = nil
+
+    def self.instance
+      @@instance ||= determine_backend_class.new
+    end
+
+    def self.determine_backend_class
+      # TODO Determine check for automate or hab or other distros
+      if license&.license_type&.downcase == "commercial"
```

Comment:
This should just be a predicate on `license` that allows for `license&.is_commercial?`

Reply:
This is not needed in this PR and removed it.